src/components/product/ServiceStatusInfo/ServiceStatusInfo.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState, useEffect, ReactNode } from 'react';
+import { Typography, Statistic, Button, Grid } from 'antd';
+import { ShrinkOutlined } from '@ant-design/icons';
+import { isUndefined, isNil } from 'lodash';
+import { PoweredBy } from './helpers/PoweredBySvg';
+import { MinimizedStatus } from './helpers/MinimizedStatus';
+import {
+  ContractsInfoContainer,
+  PoweredByLogo,
+  NextUpdateTimer,
+  OffChainContainer,
+  MobileOffChainContainer,
+} from './styles';
+
+const { Text } = Typography;
+const { Countdown } = Statistic;
+const { useBreakpoint } = Grid;
+
+type ServiceStatusInfoDetails = {
+  isHealthy?: boolean;
+  secondsLeftReceived?: number;
+  extra?: ReactNode;
+  /**
+   * extra content to be displayed on mobile size.
+   * if not defined, will use the same content as `extra`
+   */
+  extraMd?: ReactNode;
+  onTimerFinish?: () => void;
+};
+
+const DotSpace = () => <>&nbsp;&nbsp;•&nbsp;&nbsp;</>;
+
+export const ServiceStatusInfo = ({
+  isHealthy,
+  secondsLeftReceived,
+  extra,
+  extraMd,
+  onTimerFinish,
+}: ServiceStatusInfoDetails) => {
+  const screens = useBreakpoint();
+  const isMobile = (screens.xs || screens.sm) && !screens.md;
+  const [isMinimized, setIsMinimized] = useState<boolean>(false);
+  const [seconds, setSeconds] = useState<number | undefined>(0);
+
+  useEffect(() => {
+    if (!isUndefined(secondsLeftReceived)) {
+      setSeconds(secondsLeftReceived);
+    }
+  }, [secondsLeftReceived]);
+
+  const timerCountdown = isUndefined(secondsLeftReceived) ? undefined : (
+    <Countdown
+      value={Date.now() + Math.round(seconds || 0) * 1000}
+      format="ss"
+      suffix="s"
```

Comment:
We should remove the space between the number and the suffix i.e. "0s" not "0 s"